poster/styles/styles.scss

```diff
@@ -204,53 +353,67 @@ ul{
 }
 
 .link__instagram{
+  display: block;
   font-family: $font-condensed;
   font-size: 0.6rem;
   font-weight: 500;
   letter-spacing: 0.1rem;
   margin: 2rem;
   text-transform: lowercase;
+  
+  @media only screen and (min-width: 900px){
+    font-size: 0.8rem;
+  }
 }
 
+// small circles with info
+
 .info-circles{
   grid-column: 3/4;
   grid-row: 3/6;
-  font-size: 7px;
-  line-height: 9px;
+  line-height: 0.62rem;
+
+  @media only screen and (max-width: 700px){
```

Comment:
I think it's great 🙌 that you were able to keep the small circles at the top even in mobile view. Another alternative, if you wanted to make the font size larger on these circles, would be to display them horizontally for mobile:
<img width="361" alt="mobile-circles-alt" src="https://user-images.githubusercontent.com/2653980/47398793-bd8fed00-d703-11e8-8be9-79626e1abc54.png">